deploy.py

```diff
@@ -0,0 +1,252 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright 2019 SUSE LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import boto3
+import secrets
+import time
+import uuid
+
+from utils import (
+    create_kms_key,
+    create_role,
+    get_key_policy,
+    get_lambda_assume_role_policy,
+    get_lambda_ses_policy,
+    kms_encrypt,
+    update_progress
+)
+from zipfile import ZipFile
+
+request_secret = secrets.token_hex(20)
+
+function_name = input('Enter function name (git-email-hook): ') \
+                or 'git-email-hook'
+
+ses_region = input('Enter SES region name (us-east-1): ') or 'us-east-1'
```

Comment:
s/SES/deployment/ I don't think SES means much to people, well at SUSE it means SUSE Enterprise Storage so clearly a TLA problem